scanner/src/test/kotlin/scanners/fossid/FossIdTest.kt

```diff
@@ -801,7 +803,12 @@ class FossIdTest : WordSpec({
 
             val result = fossId.scan(createPackage(pkgId, vcsInfo))
 
-            val expectedAdditionalData = mapOf(SCAN_CODE_KEY to scanCode, SCAN_ID_KEY to "1")
+            val expectedAdditionalData =
+                mapOf(
+                    SCAN_CODE_KEY to scanCode,
+                    SCAN_ID_KEY to "1",
+                    SERVER_URL_KEY to "https://www.example.org/fossid"
+                )
```

Comment:
Nit: Formatting should be like

```
val expectedAdditionalData = mapOf(
    SCAN_CODE_KEY to scanCode,
    SCAN_ID_KEY to "1",
    SERVER_URL_KEY to "https://www.example.org/fossid"
)
```

But I'm also ok with keeping as-is if it passes *detekt* checks.

Reply:
You anyway have a [*detekt* issue about the test growing too large](https://github.com/oss-review-toolkit/ort/actions/runs/3966803517/jobs/6798069203#step:4:120), so you might want to address my above comment, too.

For the test size, I'd be fine with suppressing the hint by adding `@Suppress("LargeClass")` to the line before the test in the commit that starts to exceed the size.

Reply:
I have addressed your first comment. As for the LargeClass check, I see that detekt default threshold for the class size is 600 lines (and it doesn't seem to be configured in our .detekt.yml), so I put the annotation as advised.